_posts/2021-07-01-pulpcore-3.14-release-announcement.md

```diff
@@ -0,0 +1,133 @@
+ With this change, Pulp no longer needs a resource manager.
+
+If you'd like an in-depth look at the motivations, benefits, and details of the new tasking system,
+read [Matthias Dellweg's blog post](https://pulpproject.org/2021/06/21/a-new-tale/) and also watch
+ a YouTube livestreamed interview with Matthias conducted by Pulp's Brian Bouterse:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YWKw4RYluPM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Note that for the moment, this feature is in tech-preview.
+Please [let us know what you think](https://pulpproject.org/help/#github-discussions).
+
+### Sticking with the old tasking system?
+If you intend to keep using the old tasking system, you should configure the `USE_NEW_WORKER_TYPE`
+setting to `false` before upgrading. For more information about how to enable or disable the new
+tasking system, see the [documentation](https://docs.pulpproject.org/pulpcore/configuration/settings.html#use-new-worker-type).
+
+
+## New Orphan Cleanup Endpoint [#8658](https://pulp.plan.io/issues/8658)
+
+This release introduces the `/pulp/api/v3/orphans/cleanup/` endpoint.
+When you call this endpoint with `POST` and no parameters, it is the equivalent of calling.
+`DELETE /pulp/api/v3/orphans/`. Note that as part of this release,
+`DELETE /pulp/api/v3/orphans/` has been deprecated.
+
+You can specify a list of `content_hrefs` to delete using the optional parameter `content_hrefs`
+with list of content hrefs. In this case, only the specified content units will deleted by the
+orphan cleanup.
+
+## Increased Retries on Failed Downloads [#8881](https://pulp.plan.io/issues/8881)
+
+The default behavior for retrying downloads from remote sources after an error occurs has been
+updated. Downloads will now be retried on a wider range of errors, for example, HTTP 500 and
+websocket errors. The default number of retries is 3.
+
+If you're a plugin writer, you also have the option to configure the number of retries for your
+plugin with the new `DEFAULT_MAX_RETRIES` field that has to the ``Remote`` base class.
+
+## Filter the task list with correlation ID [#8891](https://pulp.plan.io/issues/8891)
+
+When using the task list endpoint, you can now filter the list by correlation ID.
+
+## Changes to Download Concurrency [#8897](https://pulp.plan.io/issues/8897)
+
+Previously, `download_concurrency` would be set to a default value upon creation. With this release,
+the default value is set to `NULL` and returns `NULL` when queried. However, a default value will
+still be used. This value comes from the Pulp global settings or can be set by the plugin writer.
+
+## Graceful shutdown for Pulpcore workers [#8930](https://pulp.plan.io/issues/8930)
+
+When a worker receives a `SIGINT` (ctrl-c) or `SIGTERM` (systemd), it will wait a short while for the
```

Comment:
```suggestion
When a worker of the distributed queueless type receives a `SIGINT` (ctrl-c) or `SIGTERM` (systemd), it will wait a short while for the
```

Maybe we can even just not mention it, since it is a change to the never before released Clueless Tasking System.
You could just add the issue to the first announcement above.

Reply:
Cool!